 imgui_tabs.cpp | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/imgui_tabs.cpp b/imgui_tabs.cpp
index bdef1be..ccb6d40 100644
--- a/imgui_tabs.cpp
+++ b/imgui_tabs.cpp
@@ -30,6 +30,8 @@
 
 
 #include "imgui_tabs.h"
+#include <iostream>
+#include <string>
 
 ImGui::ImGuiUserStyle::ImGuiUserStyle(){
     Colors[ImGuiUserCol_TabNormal] = ImVec4(0.65f, 0.65f, 0.68f, 1.00f);
@@ -103,12 +105,12 @@ void ImGui::_drawPartialRect(const ImVec2 a, const ImVec2 b, const float roundin
                 dl->PathArcToFast(ImVec2(a.x+r3-shadow_offset,b.y-r3), r3, 3, 6);
                 dl->PathArcToFast(ImVec2(a.x+r0-shadow_offset,a.y+r0), r0, 6, 9);
                 dl->PathLineTo(ImVec2(a.x,b.y));
-                dl->PathFill(col);
+                dl->PathFillConvex(col);
             } if (shadow_edges & EDGE_RIGHT){
                 dl->PathArcToFast(ImVec2(b.x-r1+shadow_offset,a.y+r1), r1, 9, 12);
                 dl->PathArcToFast(ImVec2(b.x-r2+shadow_offset,b.y-r2), r2, 0, 3);
                 dl->PathLineTo(ImVec2(b.x,b.y));
-                dl->PathFill(col);
+                dl->PathFillConvex(col);
             }
         }
     }
@@ -258,7 +260,7 @@ void ImGui::TabBar::_drawTabBarBottom() {
     ImGuiDrawContext& dc = wind->DC;
 
     const ImVec2 padding = style.WindowPadding;
-    const ImVec2 pos = ImVec2(wind->Size.x - padding.x, dc.CursorPos.y + padding.y);
+    const ImVec2 pos = ImVec2(upperLeft.x + wind->Size.x - padding.x, dc.CursorPos.y + padding.y);
     const float height = pos.y - upperLeft.y;
     dl->AddRectFilled(upperLeft, pos,ImColor(1.0f,1.0f,1.0f,0.15f),corner_rounding,ImGuiCorner_BottomLeft | ImGuiCorner_BottomRight);
     dl->AddRect(upperLeft, pos,ImColor(1.0f,1.0f,1.0f,0.15f),corner_rounding,ImGuiCorner_BottomLeft | ImGuiCorner_BottomRight);
